linters/bandit.js

```diff
@@ -53,7 +53,22 @@ module.exports = class Bandit {
    * @param {Buffer} data The raw linter results data
    */
   parseResults (data) {
-    return JSON.parse(data)
+    let parsedData = JSON.parse(data)
+
+    for (let error of parsedData.errors) {
+      let errAnnotation = {
+        filename: error.filename,
+        line_number: 1,
+        issue_severity: 'HIGH',
+        issue_confidence: 'HIGH',
+        issue_text: '',
+        test_id: 'ERROR',
+        test_name: 'Syntax error' }
```

Comment:
Put trailing } on next line to be consistent style with all the other code.

Reply:
Yes, you are right. I wonder how do you notice those things. xD